src/main/git-elegant-configure

```diff
@@ -13,6 +13,10 @@ _user_email_key="user.email"
 _user_email_default=$(git config --global "$_user_email_key")
 _user_email_message="your user email"
 
+_alias_key="alias."
+_alias_default="yes"
+_alias_message="configure aliases"
```

Comment:
`configure aliases` is not very informative. Let's use something like `add git aliases for all 'elegant git' commands`.